src/cdk-experimental/combobox/combobox.spec.ts

```diff
@@ -13,7 +13,7 @@ import {ComponentFixture, TestBed, waitForAsync} from '@angular/core/testing';
 import {By} from '@angular/platform-browser';
 import {CdkComboboxModule} from './combobox-module';
 import {CdkCombobox} from './combobox';
-import {dispatchKeyboardEvent, dispatchMouseEvent} from '@angular/cdk/testing/private';
+import {dispatchKeyboardEvent, dispatchMouseEvent} from '../../cdk/testing/private';
```

Comment:
There are a couple of similar changes. Such internal files should not be imported through a module name, but rather through a relative source import. This will help with distinguishing between public modules and internal/test-only ones.